src/components/navbar/navbar.css

```diff
@@ -12,4 +12,10 @@
 
 .dui-nav-sch-act {
   box-shadow: 0px 2px 0px 0px var(--site-bg-color);
```

Comment:
Suggestion: You can make the box-shadow into a var and use it so it's the same across the app.

Reply:
Will do it, in case I use it in more than 1 place.